### ISSUE TITLE
Terraform login provider

### ISSUE THREAD

**Issue by cedricbraekevelt** [PacoVK/tapir#432]
Hi Paco,

As I'm testing right now only the UI needs authentication and as documented in the wiki, the terraform login provider is not (yet) required.
So our modules are public as it stands, is there any timeline on when the Terraform Login provider would be implemented?

Kind regards,
Cédric

**Comment by PacoVK** [comment]
Yes only UI and management is currently protected. I could not yet estimate but for sure I can prioritize it and investigate what needs to be done.

**Comment by cedricbraekevelt** [comment]
@PacoVK  If that would be possible, that would be awesome, if you find the time that is :)
